app/uk/gov/hmrc/helptosavefrontend/config/frontendAppConfig.scala

```diff
@@ -35,7 +35,7 @@ object FrontendAppConfig extends AppConfig with ServicesConfig {
   private val contactHost = configuration.getString(s"contact-frontend.host").getOrElse("")
   private val contactFormServiceIdentifier = "MyService"
 
-  val HtsUserDetailsUrl = getConfString("help-to-save-user-details.url", "")
+  val HtsConfirmDetailsUrl = getConfString("help-to-save-confirm-details.url", "")
```

Comment:
you could use `uk.gov.hmrc.helptosavefrontend.controllers.routes.RegisterController.confirmDetails().absoluteURL()` - this would avoid needing to set the value in `application.conf`